proof-producer/tests/bin/proof-producer/test_zkevm_bbf_circuits.cpp

```diff
@@ -106,14 +108,61 @@ TEST(ProverTest, TraceIndexMismatch) {
 // Trace files contain different proto hash
 TEST(ProverTest, DifferentProtoHash) {
     const std::string trace_base_path = std::string(TEST_DATA_DIR) + "/different_proto/increment_simple.pb";
-    nil::proof_generator::Prover<ProverTests::CurveType, ProverTests::HashType> prover(
-                        ProverTests::lambda,
-                        ProverTests::expand_factor,
-                        ProverTests::max_quotient_chunks,
-                        ProverTests::grind,
+    nil::proof_generator::Prover<ProverTestBase::CurveType, ProverTestBase::HashType> prover(
+                        ProverTestBase::lambda,
+                        ProverTestBase::expand_factor,
+                        ProverTestBase::max_quotient_chunks,
+                        ProverTestBase::grind,
                         ZKEVM
     );
 
     ASSERT_TRUE(prover.setup_prover());
     ASSERT_FALSE(prover.fill_assignment_table(trace_base_path));
 }
+
+namespace fs = std::filesystem;
+
+class ProverTestsVerification : public ProverTestBase {
```

Comment:
Suspect these tests takes much time on CI (on local dev server zkevm verification with minimal size takes ~1h).
Check will be duplicated in [regular benchmarks](https://github.com/NilFoundation/placeholder/pull/309), so most probably no need add it to CI